charts/ingress-nginx/templates/controller-rolebinding.yaml

```diff
@@ -10,6 +10,7 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: {{ include "ingress-nginx.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
please remove this one. 

`namespace` should be under `metadata`

```
➜  ~ kubectl explain RoleBinding.roleRef
KIND:     RoleBinding
VERSION:  rbac.authorization.k8s.io/v1

RESOURCE: roleRef <Object>

DESCRIPTION:
     RoleRef can reference a Role in the current namespace or a ClusterRole in
     the global namespace. If the RoleRef cannot be resolved, the Authorizer
     must return an error.

     RoleRef contains information that points to the role being used

FIELDS:
   apiGroup     <string> -required-
     APIGroup is the group for the resource being referenced

   kind <string> -required-
     Kind is the type of resource being referenced

   name <string> -required-
     Name is the name of resource being referenced
```

Reply:
ok. Proves how stupid I am. Apologies for the noise. Fixing it now